hasktorch/hasktorch.cabal

```diff
@@ -138,9 +138,12 @@ test-suite spec
                     , DimnameSpec
                     , PipelineSpec
                     , Torch.Typed.AuxSpec
-                    , Torch.Typed.TensorSpec
+                    , Torch.Typed.TensorSpec0
+                    , Torch.Typed.TensorSpec1
                     , Torch.Typed.FactoriesSpec
-                    , Torch.Typed.FunctionalSpec
+                    , Torch.Typed.FunctionalSpec0
+                    , Torch.Typed.FunctionalSpec1
+                    , Torch.Typed.FunctionalSpec2
```

Comment:
I don't like this decomposition very much, but it is necessary to reduce memory at compile time.